frontend/src/components/ElectionForm/ElectionDetails.tsx

```diff
@@ -140,74 +151,111 @@ export default function ElectionDetails({ election, applyElectionUpdate, getStyl
-                            if (e.target.value == null || e.target.value == '') {
-                                applyElectionUpdate(election => { election.end_time = undefined })
-                            } else {
-                                applyElectionUpdate(election =>  
-                                election.end_time = DateTime.fromISO(e.target.value).setZone(timeZone,{keepLocalTime : true}).toJSDate())
-                            }
-                        }}
-                    />
-                    <FormHelperText error sx={{ pl: 0, mt: 0 }}>
-                        {errors.endTime}
-                    </FormHelperText>
-                </FormControl>
+
+            <Grid item xs={12} sx={{ m: 0, p: 1 }}>
+
+                <FormGroup>
+                    <FormControlLabel
+                        control={
+                            <Checkbox
+                                checked={enableStartEndTime}
+                                onChange={(e) => {
+                                    setEnableStartEndTime(e.target.checked)
+                                    if (e.target.checked) {
+                                        applyElectionUpdate(election => {
+                                            election.start_time = DateTime.now().setZone(timeZone, { keepLocalTime: true }).toJSDate()
+                                            election.end_time = DateTime.now().plus({ days: 1 }).setZone(timeZone, { keepLocalTime: true }).toJSDate()
+                                        })
+                                    }
+                                    else {
+                                        applyElectionUpdate(election => {
+                                        election.start_time = undefined
+                                        election.end_time = undefined
+                                    })
+
+                                    }
+                                }
+                                }
+                            />
+                        }
+                        label="Enable Start/End Times?" />
+                </FormGroup>
             </Grid>
+
+            {enableStartEndTime &&
+                <>
+                    <Grid item xs={4} sx={{ m: 0, p: 1 }} justifyContent='center'>
+                        <FormControl fullWidth>
+                            <InputLabel id="demo-simple-select-label">Time Zone</InputLabel>
+                            <Select
+                                labelId="demo-simple-select-label"
+                                id="demo-simple-select"
+                                value={timeZone}
+                                label="Time Zone"
+                                onChange={(e) => applyElectionUpdate(election => { election.settings.time_zone = e.target.value })}
+                            >
+                                <MenuItem value={timeZone}>{timeZone}</MenuItem>
```

Comment:
nit

If timezone is in the timezones list would there be duplicate timezones in the list? 

I think that's probably ok. When I'm entering my country in a form I never know if they'll have USA on top for convenience or if I should search for it alphabetically. So I think having duplicates might be a better experience

That said, is it possible to add a divider to the list? That way we can communicate that the first item in the list has some significance

Reply:
Yes, whatever is currently selected will be at the top of the list. But thinking about that more I think the value at the top of the list should be the local time zone. Or we could have both current and local at the top. A divider is a good idea, I'll need to see if that is possible.

Reply:
Changed top of list to local time zone and added divider.